### PR TITLE
Fix Bug 1477766 - Display Simplified Onboarding Tour Cards in specific order

### PR DESCRIPTION
**Pull request by sarracini**
Adds an order property to the message, and sorts on that order. Defaults to 0 if no order provided - meaning this bundle doesn't care about this order

### CODE REVIEW

File: lib/OnboardingMessageProvider.jsm [hunk 1]
```diff
@@ -50,11 +53,12 @@ const ONBOARDING_MESSAGES = [
     id: "ONBOARDING_4",
     template: "onboarding",
     bundled: 3,
+    order: 1,
     content: {
-      title: "Extensions",
-      text: "Make browsing faster, smarter, or safer with browser apps. Protect passwords, find deals, download videos, and much more. You can even block annoying ads with extensions like Ghostery.",
+      title: "Block Ads with Ghostery",
```
Comment (sarracini) [inline lib/OnboardingMessageProvider.jsm]: Some copy changes to the Ghostery extension that were requested prior to 62

File: content-src/asrouter/schemas/provider-response.schema.json [hunk 2]
```diff
@@ -21,6 +21,14 @@
             "description": "An id matching an existing Activity Stream Router template",
             "enum": ["simple_snippet"]
           },
+          "bundled": {
+            "type": "integer",
+            "description": "The number of messages of the same template this one should be shown with (optional)"
+          },
+          "order": {
```
Comment (k88hudson) [inline content-src/asrouter/schemas/provider-response.schema.json]: can you add to the docs as well?

File: lib/ASRouter.jsm [hunk 3]
```diff
@@ -283,8 +283,12 @@ class _ASRouter {
     return message;
   }
 
+  _orderBundle(bundle) {
+    return bundle.sort((a, b) => a.order > b.order);
```
Comment (k88hudson) [inline lib/ASRouter.jsm]: This should be `a.order - b.order`

File: content-src/asrouter/schemas/provider-response.schema.json [hunk 4]
```diff
@@ -21,6 +21,14 @@
...
+            "type": "integer",
```
Comment (k88hudson) [inline content-src/asrouter/schemas/provider-response.schema.json]: Should this maybe have `"minimum": 0` as well or are we allowing negative values?

**Pull request merged**